sdk/typescript/dev/src/index.ts

```diff
@@ -17,21 +11,21 @@ class TypescriptSdkDev {
   project: Container
 
   constructor(source: Directory) {
-    // Extract package.json and yarn.lock to a temporary directory
+    // Extract packageon and yarn.lock to a temporary directory
     const dependencyFiles = dag
       .directory()
-      .withFile("package.json", source.file("package.json"))
+      .withFile("packageon", source.file("packageon"))
       .withFile("yarn.lock", source.file("yarn.lock"))
 
     // Get source without generated files nor useless files.
     const sourceCode = dag.directory().withDirectory("/", source, {
       include: [
         "**/*.ts",
-        "tsconfig.json",
-        "package.json",
+        "tsconfigon",
+        "packageon",
         "yarn.lock",
-        ".mocharc.json",
-        "eslint.config.js",
+        ".mocharcon",
+        "eslint.config",
```

Comment:
That's because you have the tooling installed locally right? This module was supposed to be able to develop the SDK without having anything other than dagger installed. Think about someone else that doesn't usually work with TypeScript but needs to fix something.

However if it's just gathering dust and not working than there's no point, but ideally it would be a dependency of `.dagger` and that would call out to this module for implementation. Of course, that's only useful if this module can do a few more tasks, not just those ones.

See Python, for example:

https://github.com/dagger/dagger/blob/55fce3dfd68963b54f6eaa74a229a3ca8d22c3af/.dagger/sdk_python.go

Reply:
I'll take care of that in a follow up then!